lineman/app/components/thread_page/comment_form/comment_form.coffee

```diff
@@ -3,15 +3,17 @@ angular.module('loomioApp').directive 'commentForm', ->
   restrict: 'E'
   templateUrl: 'generated/components/thread_page/comment_form/comment_form.html'
   replace: true
-  controller: ($scope, FlashService, Records, CurrentUser) ->
+  controller: ($scope, FormService, Records, CurrentUser) ->
     group = $scope.comment.discussion().group()
     discussion = $scope.comment.discussion()
 
-    $scope.submit = ->
-      $scope.comment.save().then ->
-        $scope.comment = Records.comments.build(discussion_id: discussion.id)
-        $scope.$emit('commentSaveSuccess')
-        FlashService.success('comment_form.messages.created')
+    applyFormService = (=>
```

Comment:
Ah, good know, although looking at this again, I don't think I need to name this function, I can just say `$scope.submit = FormService.submit`

Before I was trying to do something where I called applyFormService in the success callback to reset the comment, but didn't work.